src/server/apsystem.ts

```diff
@@ -471,6 +474,20 @@ export default class ActivityPubSystem {
     }))
   }
 
+  async notifyInteracted (fromActor: string, activity: APActivity): Promise<void> {
+    const interacted = await this.store.forActor(fromActor).interacted.list()
+    // loop through each
+    await Promise.all(interacted.map(async (mention) => {
+      try {
+        const actorURL = await this.mentionToActor(mention)
+        return await this.sendTo(actorURL, fromActor, activity)
+      } catch (e) {
+        // TODO: Remove deleted accounts?
+        console.error(`Unable to notify actor ${fromActor}`, e)
```

Comment:
apsystem has a logging implementation now, can you change it to this.log's equivalent?